### ISSUE TITLE
Handle Notification from Terminated state

### ISSUE THREAD

**Issue by xuanzi23** [das08/PushNotificationTest#1]
I would like to ask about did you project can handle notification from terminated state?
Thanks

**Comment by das08** [comment]
No, I could not receive silent push notification when I terminate my app intentionally.
Though sometimes notification observer fires after I re-launched the app. I guess you need to send push notification (not silent one) if you want to wake up terminated app.
